### PR TITLE
ryujinx: 1.0.6574 -> 1.0.6659

### PR DESCRIPTION
**Pull request by IvarWithoutBones**
###### Motivation for this change
Decided to write an update script, so hopefully r-ryantm can update this as well.

###### Things done

- [X] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by MGThePro — APPROVED** [review]

File: pkgs/misc/emulators/ryujinx/updater.sh [hunk 1]
```diff
@@ -1,10 +1,33 @@
-#!/usr/bin/env nix-shell
-#!nix-shell -i bash -p curl jq common-updater-scripts dotnet-sdk_5
+#! /usr/bin/env nix-shell
+#! nix-shell -i bash -p common-updater-scripts nix-prefetch-git jq dotnet-sdk_5
```
Comment (SuperSandro2000) [inline pkgs/misc/emulators/ryujinx/updater.sh:2]: grep and gnused are missing.

Comment (IvarWithoutBones) [reply pkgs/misc/emulators/ryujinx/updater.sh]: Done :+1:

Comment (SuperSandro2000) [reply pkgs/misc/emulators/ryujinx/updater.sh]: You are missing gnused.

Comment (IvarWithoutBones) [reply pkgs/misc/emulators/ryujinx/updater.sh]: Not sure how i forgot about that, but it should be in there now :)

**Comment by IvarWithoutBones** [comment]
I've updated the package once again, and added an missing dependency it complained about at runtime. Should be ready to merge now @SuperSandro2000

**Pull request closed without merging**